thirdparty_containers/system-test/dockerfile/system-test.sh

```diff
@@ -0,0 +1,69 @@
+#/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#Set up Java to be used by the system-test
+
+if [ -d /java/jre/bin ];then
+	echo "Using mounted Java8"
+	export JAVA_BIN=/java/jre/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+	java -version
+elif [ -d /java/bin ]; then
+	echo "Using mounted Java9"
+	export JAVA_BIN=/java/bin
+	export JAVA_HOME=/java
+	export PATH=$JAVA_BIN:$PATH
+	java -version
+else
+	echo "Using docker image default Java"
+	java_path=$(type -p java)
+	suffix="/java"
+	java_root=${java_path%$suffix}
+	export JAVA_BIN="$java_root"
+	echo "JAVA_BIN is: $JAVA_BIN"
+	$JAVA_BIN/java -version
+fi
+
+TEST_SUITE=$1
+
+echo "PATH is : $PATH"
+echo "JAVA_HOME is : $JAVA_HOME"
+echo "type -p java is :"
+type -p java
+echo "java -version is: \n"
+java -version
+
+cd /openjdk-tests
+
+./get.sh -t /openjdk-tests -p x64_linux -i openj9
+
+cd TestConfig
+
+export SPEC=linux_x86-64_cmprssptrs
+export BUILD_LIST=systemtest
```

Comment:
Do we set JDK_IMPL?

Reply:
Not sure if  we need to set JDK_IMPL, wouldn't it pick from the playlist. For mauve tests, I have not specified any impl in the playlist as they are applicable to both hotspot and openj9. But for shareclasses I have specified openj9 as impl.

Reply:
- `JDK_IMPL` value should be set based on the SDK that we are testing. The playlist checks `JDK_IMPL` value to determine if the test is applicable or not.

   If `JDK_IMPL` is not provided, the program will use the default value `openj9`
https://github.com/eclipse/openj9/blob/5839468bd3fe2217b7590eb27368c2c246157f04/test/TestConfig/run_configure.mk#L61-L63

- Actually, to make the story simple, can we just mount test material directly when running docker? Similar to how to mount java. In this way, there is no need to run `get.sh` in this script.

Reply:
Mounted material would need to be updated regularly, I would rather prefer moving get.sh and compilation part into the docker image and just have the running part in shell script. 

@smlambert @Mesbah-Alam - Any thoughts?

Reply:
> Mounted material would need to be updated regularly

No. Test material is checked out during every build. We do not need to update it at all. This is the same idea as you mount jdk in playlist.xml, something like:

```
docker run -v $(JDK_HOME):/java -v <path to test root folder>:/test <docker_image_name> sanity.system
```

Since we are passing in repo, branch and sha for regular build and Grinder, it is all been taking care of by mounting the existing test material. If we choose to use run `get.sh` again in docker, not only we git clone twice, but also we need to get repo, branch and sha parameters from the build and pass them into `get.sh` in docker.